doc/gpo/PowerToys.admx

```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.
+     Licensed under the MIT License. -->
+<policyDefinitions xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" revision="1.0" schemaVersion="1.0" xmlns="http://schemas.microsoft.com/GroupPolicy/2006/07/PolicyDefinitions">
+  <policyNamespaces>
+    <target prefix="powertoys" namespace="Microsoft.Policies.PowerToys" />
+    <using prefix="windows" namespace="Microsoft.Policies.Windows" />
+  </policyNamespaces>
+  <resources minRequiredRevision="1.0" />
+  <categories>
+    <category name="PowerToys" displayName="$(string.PowerToys)">
+      <parentCategory ref="windows:WindowsComponents" />
+    </category>
+  </categories>
+  <policies>
+    <policy name="ConfigureEnabledUtilityAlwaysOnTop" class="Machine" displayName="$(string.ConfigureEnabledUtilityAlwaysOnTop)" explainText="$(string.ConfigureEnabledUtilityDescription)" key="Software\Policies\Microsoft\Windows\PowerToys" valueName="ConfigureEnabledUtilityAlwaysOnTop">
+      <parentCategory ref="PowerToys" />
+      <supportedOn ref="windows:SUPPORTED_Windows_10_0_RS5" />
```

Comment:
Makes no sense. We should write the PowerToys version here.

Reply:
> @jaimecbernardo, @crutkas Why do we only support computer scoope? Is there any reason I don't see at the moment?
> 
> I think we should support both and give the computer scoope mir priority.

The main reason for doing it at the machine level was for sysadmins to set it for their org computers.
What's the purpose of having these be User level?

Reply:
You mean the version supported by PowerToys? 😅 That makes sense

Reply:
Yes